xml_struct_tests/ui/lib.rs

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
These build tests are currently incomplete. Missing tests include fields of types which don't implement the appropriate traits, fields of types which don't implement the appropriate traits but _do_ have identically-named methods, and probably a battery of tests to validate that all of our happy-path cases compile properly (although this is kind of handled by the integration tests as well).